entity-framework/core/get-started/xamarin.md

```diff
@@ -0,0 +1,197 @@
+
+Click **Add** in the toolbar.
+
+A page then appears that allows you to fill out information about a new blog post.
+
+![Screenshot of add new post page](_static/xamarin-tutorial-4.png)
+
+Fill out all the information and click **Save** in the toolbar.
+
+The new post will be associated to the blog post you clicked on in a previous step and will save to the app's SQLite database and show in the list.
+
+Go back to the blog list page. And click **Delete All** in the toolbar. All blogs and their corresponding posts will then be deleted from the app's SQLite database.
+
+![Screenshot of app with all blogs deleted](_static/xamarin-tutorial-5.png)
+
+## Explore the code
+
+The following sections will walk you through the code in the sample project that reads, creates, updates, and deletes data from a SQLite database using EF Core with Xamarin.Forms.
+
+It is assumed that you are familiar with the Xamarin.Forms topics of displaying data and navigating between pages.
+
+## Entity Framework Core NuGet packages
+
+To create Xamarin.Forms apps with EF Core, you install the package for the EF Core database provider(s) you want to target into all of the projects in the Xamarin.Forms solution. This tutorial uses SQLite.
+
+The following NuGet package is needed in each of the projects in the Xamarin.Forms solution.
+
+* `Microsoft.EntityFrameworkCore.Sqlite`
+
+## Model classes
+
+Each table in the SQLite database accessed through EF Core is modeled in a class. In this sample, two classes are used: `Blog` and `Post` which can be found in the `Models` folder.
+
+The model classes are composed only of properties, which model columns in the database.
+
+* **Blog.cs**
+
+  [!code-csharp[](../../../samples/core/Xamarin/EFGetStarted/Models/Blog.cs)]
+
+* The `Posts` property defines a parent-child relationship between `Blog` and `Post`.
+
+* **Post.cs**
+
+  [!code-csharp[](../../../samples/core/Xamarin/EFGetStarted/Models/Post.cs)]
+
+* The `BlogId` and `Blog` properties relate back to the parent `Blog` object for the instance of the `Post`.
+
+## Data context
+
+The `BloggingContext` class is located in the `Services` folder and inherits from the EF Core `DbContext` class. A `DbContext` is used to group together database queries and changes.
+
+  [!code-csharp[](../../../samples/core/Xamarin/EFGetStarted/Services/BloggingContext.cs)]
+
+* Both properties in this class of type `DbSet` are used to operate on the underlying tables representing Blogs and Posts.
+* The `SQLitePCL.Batteries_V2.Init()` is needed in the constructor to initiate SQLite on iOS.
+* The `OnConfiguring` function sets up the location of the SQLite database on the physical device.
+
+## Create, read, update & delete
+
+The following are some instances in the app where EF Core is used to access SQLite.
+
+### Read
+
+* Return all records.
+  * The `OnAppearing` function of `BlogsPage.xaml.cs` returns all `Blog` records and stores them into a `List` variable.
+
+```c-sharp
```

Comment:
Almost there! For syntax highlighting, these should be three back-ticks then `csharp` not `c-sharp` (no hyphen).